proto/hack/fake-gen/main.go

```diff
@@ -14,8 +14,8 @@ import (
 	"strings"
 
 	. "github.com/dave/jennifer/jen"
-	plugin "github.com/golang/protobuf/protoc-gen-go/plugin"
 	"google.golang.org/protobuf/proto"
+	plugin "google.golang.org/protobuf/types/pluginpb"
 )
```

Comment:
Most impactful change is here. The file generated from this code has no changes when run, so I'm confident nothing was broken here.